docs/conf/portland/2018/cfp.rst

```diff
@@ -0,0 +1,105 @@
+from underrepresented groups within the tech community.
+
+We prefer talks backed by experience and experimentation to talks about theory,
+and we definitely don't like talks that bad-mouth technologies or approaches.
+Don't tell us why you hate something – tell us how you overcame the problems it was causing.
+
+We tend to avoid talks about specific tools, which often turn into marketing
+pitches or tutorials. We would much rather hear about process, culture, data,
+people, or the metaphysical side effects of spending your life thinking about docs.
+
+When crafting talk proposals remember that you're going to be talking to a mix
+of levels of expertise, skill sets, and professions. Your talk doesn't have to be
+relevant to everyone, but it should be relevant to most people and shouldn't
+make too many assumptions about what people already know.
+
+Take a look at the abstracts for accepted talks from the last couple of
+conferences for some ideas:
+
+* `Portland 2017 <http://www.writethedocs.org/conf/na/2017/speakers/>`_
+* `Portland 2016 <http://www.writethedocs.org/conf/na/2016/speakers/>`_
+* `Prague 2017 <http://www.writethedocs.org/conf/eu/2017/speakers/>`_
+* `Prague 2016 <http://www.writethedocs.org/conf/eu/2016/speakers/>`_
+
+Need help?
+-----------
+
+If you need a hand preparing your talk proposal, there are lots of good places to start:
+
+* **Community mentorship** – We have an ever-growing pool of previous Write the Docs speakers, many of whom are happy to be a second pair of eyes on talk proposals. If you're interested in working with a past speaker, let us know!
+* **Meetup brainstorming** – For some in-person workshopping, check in on your `local meetup group<http://www.writethedocs.org/meetups/>`_ and see if they have a talk brainstorming session on their calendar. If they don't... ask if they're planning one!
+* **Slack hivemind** – You can also hit up the hivemind directly on the Write the Docs Slack, any time of day! (If you're not registered yet, you can at `http://slack.writethedocs.org/<http://slack.writethedocs.org/>`_.)
+
+Presentation format
+-------------------
+
+Presentations will be scheduled in 30-minute blocks. We won't be doing Q&A after
+the talks, but we ask that speakers be available at the front of the stage
+for a few minutes of the break after their talk to answer questions.
+
+Speaker benefits & logistics
+----------------------------
+
+If you are selected to speak at Write the Docs, we will waive your attendance
+fee. We regret that we are not able to cover the cost of travel or lodging for
+all speakers at this time. By not covering travel costs for our speakers, we are
+able to keep ticket prices low and to make the conference accessible to the
+largest number of participants possible. If your proposal is accepted but you
+are unable to attend due to travel costs, please let us know and we will do our
+best to help make it happen.
```

Comment:
I think this can maybe be worded differently. I think more of a "We don't cover costs for all our our speakers to keep prices lower, but have a travel assistance fund for people who need it". I feel like we bury the lede a bit.